docs/VictoriaLogs/FAQ.md

```diff
@@ -175,3 +175,12 @@ And for the following log, its `_msg` will be `foo bar in body`:
   "body": "foo bar in body"
 }
 ```
+
+## What is the max size of a log record?
+
+The max length of a single log record the VictoriaLogs is capable of handling
```

Comment:
The 2MB is the max size of a log record VictoriaLogs can handle because it is limited by the max block size. Log records of such sizes aren't handled efficiently by VictoriaLogs because per-block overhead translates to a single log record, and this overhead is big. VictoriaLogs works optimally with log records of up to 10KB. It works OK with log records of up to 100KB. It works not so optimal with log records exceeding 100KB.